boards/shared/Src/Io/Io_SharedFrequencyOnlyPwmInput.c

```diff
@@ -0,0 +1,127 @@
+#include <main.h>
```

Comment:
nit; `main.h` is not a standard library we should should use `"main.h"` rather than `<main.h>`

Reply:
fixed

Reply:
fixed